compiler/optimizations/deadCodeElimination.cpp

```diff
@@ -322,37 +322,25 @@ static bool isDeadStringOrBytesLiteral(VarSymbol* string) {
 //
 //   def  new_temp  : string; // defTemp
 //
-//   def  ret  : string;  // defFactoryTemp
-//   call init(ret);  // defaultInit
-//   call initWithBorrowedBuffer(ret, c"literal string", ...);  // factoryCall
-//   move new_temp, ret   //tmpMove
+//   call createStringWithBorrowedBuffer(ret, c"literal", ...); // factoryCall
```

Comment:
I think this comment should pass 'new_temp' instead of 'ret'

Reply:
good catch!